evalml/tests/automl_tests/test_automl.py

```diff
@@ -3641,48 +3742,48 @@ def test_automl_respects_pipeline_custom_hyperparameters_with_duplicate_componen
             "Random Forest Classifier": {"n_estimators": Categorical([50, 100])},
         }
         component_graph = {
-            "Imputer": ["Imputer"],
-            "Imputer_1": ["Imputer", "Imputer"],
-            "Random Forest Classifier": ["Random Forest Classifier", "Imputer_1"],
+            "Name_dict": {
+                "Imputer": ["Imputer"],
+                "Imputer_1": ["Imputer", "Imputer"],
+                "Random Forest Classifier": ["Random Forest Classifier", "Imputer_1"],
+            }
         }
-        pipeline_ = BinaryClassificationPipeline(
-            component_graph, custom_name="Pipeline from dict"
-        )
 
     automl = AutoMLSearch(
         X,
         y,
         problem_type="binary",
-        allowed_pipelines=[pipeline_],
+        allowed_component_graphs=[component_graph],
         custom_hyperparameters=custom_hyperparameters,
         max_batches=5,
     )
+
     automl.search()
     for i, row in automl.full_rankings.iterrows():
         if "Mode Baseline Binary" in row["pipeline_name"]:
             continue
-        if row["pipeline_name"] == "Pipeline Dict":
-            assert row["parameters"]["Imputer"]["numeric_impute_strategy"] in {
-                "most_frequent",
-                "mean",
-            }
+        if row["pipeline_name"] == "Name_linear":
+            assert row["parameters"]["Imputer"]["numeric_impute_strategy"] == "mean"
```

Comment:
What's causing this diff? Is it the deletion of `_frozen_pipeline_parameters`?

Reply:
Yes I think so